doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+The R and Python programming languages [@R; @Python] and the following R and Python packages were used to perform the analysis:  docopt [@docopt],  knitr [@knitr], tidyverse [@tidyverse], infer[@infer], docopt [@docoptpython]. The code used to perform the analysis and create this report can be found here: https://github.com/UBC-MDS/DSCI_522_inference_on_indigenous_vs_non_indigenous_sentence_length_differences.
+
+
+# Results and discussion 
+
+Firstly, the only relevant features for our hypothesis test were the ‘aggregate sentence length’ and the ‘racial grouping column’. We decided to use the ‘racial grouping’ column over the race column as it already contained two groups as required by the hypothesis testing: ‘indigenous’ and ‘non-indigenous’. All other columns were dropped. The count of observations in either group was `r COUNT_IND` and `r COUNT_NON_IND`, meaning we had a large enough sample size to carry out a t-test if need be. 
+
+```{r observations_group, echo=FALSE}
+knitr::kable(model_quality$table, caption = "Table 1. Number of observations in each group.")
+```
+
+
+To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (COLORS) using a density plot t. As the distribution for both groups was right skewed, we decided to use the median as our parameter as it is less affected by extreme values as It based on the count.  
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 1. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+Looking at the boxplot, we can see that there is a large overlap in the confidence intervals between the two groups, and potentially indicates that there is not a significant. 
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 2. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+As we selected the median, we subsequently carried out a permutation test under the null hypothesis computationally with `r N_REPEATS`.  This model is applicable as we have a large sample size.
```

Comment:
We have a large enough sample size to apply a permutation test. We carried out the permutation test  with `r N_REPEATS` under the null hypothesis, computing and comparing the median for each of the two groups.